sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/Identity.java

```diff
@@ -482,9 +475,8 @@ public void run() {
 
     @UiThread
     private synchronized void signInInteractively() {
-        if (mActivity != null) {
-            AppCenterLog.info(LOG_TAG, "Signing in using browser.");
-            mSignInDelayed = false;
+    	if (mAuthenticationClient != null && mActivity != null) {
+    		AppCenterLog.info(LOG_TAG, "Signing in using browser.");
```

Comment:
Please commit with Android Studio:

<img width="290" alt="Screen Shot 2019-03-12 at 11 52 09 AM" src="https://user-images.githubusercontent.com/623672/54228816-367bbf80-44c0-11e9-8b13-8c31a21bd897.png">

I can tell formatting is not right here ;)